kong/plugins/response-ratelimiting/policies/init.lua

```diff
@@ -208,8 +208,13 @@ return {
     usage = function(conf, identifier, name, period, current_timestamp)
       local red = redis:new()
       red:set_timeout(conf.redis_timeout)
-
-      local ok, err = red:connect(conf.redis_host, conf.redis_port)
+      -- use a special pool name only if redis_database is set to non-zero
```

Comment:
The below condition isn't "non-zero", but "non-nil". Either one is probably fine, there is no harm in always specifying a pool name.